src/App.js

```diff
@@ -3,11 +3,15 @@ import Navbar from './containers/Navbar/Navbar';
 import Home from './containers/Home/Home';
 
 class App extends Component {
+  state = {
+    textBool: true
+  }
```

Comment:
Normally, states like this would either be placed in the Redux store in OR inside the React's new Context API, but for now, we'll place the `textBool` state here. But `textBool` seems like a very vague naming convention. I would rather name this state to something like: `showTabMessage`.